encoding/ascii85.ts

```diff
@@ -65,6 +65,8 @@ const Z85 =
  * @param [options.delimiter] whether to use a delimiter, if supported by encoding standard
  */
 export function encode(uint8: Uint8Array, options?: Ascii85Options): string {
+  // Create a new Uint8Array from the given uint8 instead of using it
+  uint8 = new Uint8Array(uint8);
```

Comment:
Can you try to fix the line `const view = new DataView(uint8.buffer);` instead of creating new uint8 array here. This causes unnecessary copy of underlying ArrayBuffer, and makes the function slower.

I think that line should be something like `new DataView(uint8.buffer, uint8.byteOffset, uint8.byteLength)`.

Reply:
Thanks for your suggestion!